iree/turbine/kernel/wave/utils.py

```diff
@@ -1235,6 +1235,20 @@ def _get_start_indices(
     return start_indices
 
 
+def _get_fastest_index(indices: dict[IndexExpr, IndexSequence]):
```

Comment:
nit: as this is now a public function, we can remove underscore from name.

Reply:
sg!

Reply:
thanks for the catch, done! :)